content/ko/docs/faq/_index.md

```diff
@@ -1,498 +1,544 @@
+그 위치를 재정의(override)할 수 있는 환경변수를 사용할 수 있습니다.
+
+- `$XDG_CACHE_HOME`: 캐시 파일 보관 장소를 다른 곳으로 설정
+- `$XDG_CONFIG_HOME`: 헬름 설정 파일 보관 장소를 다른 곳으로 설정
+- `$XDG_DATA_HOME`: 헬름 데이터 보관 장소를 다른 곳으로 설정
+
+기존 저장소가 있는 경우, `helm repo add...`를 사용하여 저장소를 다시 추가해야 합니다.
+
+## 삭제
+
+### 로컬 헬름을 삭제하고 싶어요. 그 파일들은 모두 어디에 있나요?
+
+헬름은 `helm` 바이너리 파일과 함께 일부 파일을 다음 위치에 저장합니다.
+
+- $XDG_CACHE_HOME
+- $XDG_CONFIG_HOME
+- $XDG_DATA_HOME
+
+다음 표에서는 OS 별로 각 항목에 대한 기본 폴더를 제공합니다.
+
+| 운영 체제        | 캐시 경로                   | 설정 경로                        | 데이터 경로               |
+|------------------|-----------------------------|----------------------------------|---------------------------|
+| 리눅스           | `$HOME/.cache/helm `        | `$HOME/.config/helm `            | `$HOME/.local/share/helm` |
+| 맥OS             | `$HOME/Library/Caches/helm` | `$HOME/Library/Preferences/helm` | `$HOME/Library/helm `     |
+| 윈도우           | `%TEMP%\helm  `             | `%APPDATA%\helm `                | `%APPDATA%\helm`          |
+
+## 트러블슈팅
+
+### GKE (구글 컨테이너 엔진)에서 "현재 열려 있는 SSH 터널이 없습니다"라고 나와요
+
+```
+Error: Error forwarding ports: error upgrading connection: No SSH tunnels currently open. Were the targets able to accept an ssh-key for user "gke-[redacted]"?
+```
+
+오류 메시지의 또 다른 변형판은 다음과 같습니다.
+
+```
+Unable to connect to the server: x509: certificate signed by unknown authority
+```
+
+해당 이슈는 로컬 쿠버네티스 설정 파일에 올바른 인증서가 있어야 한다는
+것입니다.
+
+GKE에 클러스터를 생성하면 SSL 인증서 및 인증 기관을 포함한
+인증서가 제공됩니다. 이러한 파일은 `kubectl`과 `helm`이
+접근할 수 있도록 쿠버네티스 설정 파일에 저장되어야
+합니다(기본값: `~/.kube/config`).
+
+### 헬름 2에서 마이그레이션한 후, `helm list`에는 릴리스들이 일부만 보여요(또는 안 보여요).
+
+헬름 3는 이제 클러스터 네임스페이스를 사용하여 릴리스들을 구획한다는 사실을 깜빡하신 것 같습니다.
+릴리스를 참조하는 모든 명령어는 다음 중 하나로 수행해야 합니다.
+
+* 활성 쿠버네티스 컨텍스트의 현재 네임스페이스에 의존합니다(`kubectl config view --minify` 명령어로 확인).
+* `--namespace`/`-n` 플래그를 사용하여 올바른 네임스페이스를 지정합니다.
+* `helm list` 명령어에 대해서는 `--all-namespaces`/`-A` 플래그를 지정합니다.
+
+이는 릴리스를 참조하는 `helm list`, `helm uninstall` 및 기타 모든 `helm` 명령어에 적용됩니다.
+
+### 맥OS에서는 `/etc/.mdns_debug` 파일에 접근합니다. 왜 그런가요?
+
+맥OS에서는 헬름이 `/etc/.mdns_debug`라는 파일에 접근하려고
+하는 것으로 알려져 있습니다. 파일이 있는 경우, 헬름은 파일이 실행되는 동안
+파일 핸들을 열어 둡니다.
+
+이 문제는 맥OS의 MDNS 라이브러리로 인해 발생합니다.
+디버깅 설정을 읽기 위해 MDNS 라이브러리를 로드하려고 시도합니다(활성화된 경우).
+파일 핸들을 열지 말아야 하며, 이 문제는 애플(Apple)에도 보고되었습니다.
+다만, 이러한 동작을 일으키는 것은 헬름이 아닌 맥OS입니다.
+
+헬름이 이 파일을 로드하지 않도록 하려면 호스트 네트워크 스택을 사용하지 않는
+정적 라이브러리로서 헬름을 컴파일하면 됩니다. 이렇게 하면 헬름의 바이너리 크기가
+커지기는 하지만, 그 파일이 열리는 것은 막을 수 있습니다.
+
+이 문제는 원래 잠재적인 보안 문제로 분류되기도 했습니다.
+그러나 이후 이 동작으로 인한 결함이나 취약점은 없는 것으로 판명되었습니다.
+
+### helm repo add가 동작한 후 실패합니다.
+
+헬름 3.3.1 및 이전 버전에서 이미 존재하는 저장소(repo)를 추가하려고 하면
+`helm repo add <reponame> <url>` 명령어를 실행해도 출력되는 내용은 없습니다.
+`--no-update` 플래그를 사용하면 저장소가 이미 등록되어 있는 경우 오류를 발생시킵니다.
+
+헬름 3.3.2 이상에서 기존 저장소를 추가하려고 하면 다음 오류가 발생합니다.
+
+이제 기본 동작은 반대가 됩니다. `--no-update`는 이제 무시되며,
+기존 저장소를 교체(덮어쓰기)하려면 `--force-update`를 사용할 수 있습니다.
+
+이는 [헬름 3.3.2 릴리스 노트](https://github.com/helm/helm/releases/tag/v3.3.2)에서
+설명된대로 보안 픽스에 따라 단절적 변경(breaking change)이 있었기 때문입니다.
```

Comment:
We added one more section here: https://helm.sh/docs/faq/#enabling-kubernetes-client-logging